src/main/resources/reference.conf

```diff
@@ -1674,6 +1677,35 @@ resourceTypes = {
     allowLeaving = false
     reuseIds = true
   }
+
+  cbas-submission = {
+    actionPatterns = {
+      abort = {
+        description = "Abort this cbas-submission"
+      }
+      read = {
+        description = "read information about the submission"
+      }
+      create_with_parent = {
+        description = "Enables creating the request object with a parent"
+      }
+      read_creator = {
```

Comment:
This resource action will allow those who have it to be able to get the resource creator's email. This is similar to `read_policies`, where users are allowed to get the emails of people in policies on a resource.